resources/views/install/agreement.blade.php

```diff
@@ -0,0 +1,96 @@
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    @include('../_partials._html_header', $data)
+    @php
+      $currentPage
+    @endphp
+    <body class="bg-dark text-light container-fluid" data-bs-theme="dark">
+        <div class="row">
+            <div class="col-sm-3">
+                <div class="d-flex flex-nowrap">
+                  <div class="d-flex flex-column flex-fill flex-shrink-0 p-3 text-bg-dark">
+                    <div class="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-white text-decoration-none">
+                        <div class="text-center">
+                            <img class="" src="/img/badgerstaffpanel-logo.png" width="64" height="64" alt="website-logo">
+                            <h5>Installation Script</h5>
+                        </div>
+                    </div>
+                    <hr>
+                    <ul class="nav nav-pills flex-column"> 
+                      <li class="nav-item"><a href="welcome" class="nav-link text-white"><i class="bi text-success bi-check2-square me-2"></i> Welcome</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link active" aria-current="page"><i class="bi bi-square me-2"></i> Agreements</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Configuration</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Database</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Administrative User</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Discord</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Groups</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Complete</a></li>

```

Comment:
If the links just go to the current URL, do they even need to be pointed to anything or exist at all?